hack/community/generate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Generate the new community manifests, given the current wmco manifests.
+
+# Must be ran within community-operators-prod root.
+
+# Usage: community.sh <new_community_wmco_version>
+# Example:
+# Update community manifests v5.0.1
+# Run: bash ../windows-machine-config-operator/hack/community/generate.sh 5.0.1
+
+# create and copy manifests to new folder and rename
+now=`date +"%Y-%m-%d"`
```

Comment:
[createdAt](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/community-windows-machine-config-operator/5.0.0/manifests/community-windows-machine-config-operator.5.0.0.clusterserviceversion.yaml#L10) format should match: YYYY-MM-DD T HH:MM:SS Z